### PR TITLE
Fixes #24670: Improve 8.1 release notes

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/24670

### CODE REVIEW

File: release-data/changelogs/8.1/index.adoc [hunk 1]
```diff
@@ -150,8 +156,8 @@ were reworked. For most used commands, the changes are:
 * `rudder package plugin enable/disable <plugin>` -> `rudder package enable/disable <plugin>`
 * `rudder package check-connection` -> `rudder package update --check`
 
-You can still use the previous implementation with `RUDDER_PKG_COMPAT=1 rudder package ...`, but it will
-be removed in an upcoming release.
+To ease the transition, can still use the previous implementation with `RUDDER_PKG_COMPAT=1 rudder package ...`,
```
Comment (ElaadF) [inline release-data/changelogs/8.1/index.adoc:159]: you*

Comment (ElaadF) [inline release-data/changelogs/8.1/index.adoc:159]: Is it correct without `you` before `can`

Comment (amousset) [reply release-data/changelogs/8.1/index.adoc:159]: definitely not

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ElaadF — APPROVED** [review]

**Pull request merged**